src/app/app.page/app.page.component.html

```diff
@@ -448,12 +448,12 @@ <h4 class="card-title text-left" i18n="@@newLiveStreamCardTitle">
                                                     <div style="font-size: 1.1em">
                                                         <!-- Play Button -->
                                                         <button *ngIf="row.status=='broadcasting' && (appSettings.webRTCEnabled == true|| appSettings.hlsMuxingEnabled == true) "
-                                                                data-toggle="tooltip" i18n-title title="Play Stream"
+                                                                data-toggle="tooltip" i18n-title title="Watch Stream"
                                                                 (click)="playLive(row.streamId)"
                                                                 class="btn btn-success btn-simple btn-magnify btn-xs big-icons"
                                                                 type="button">
                                                             <span class="btn-label">
-                                                                <i class="ti-control-play"></i>
+                                                                <i class="ti-eye"></i>
```

Comment:
Please don't use eye icon for playing. Because Play icon is universal and we shouldn't change this behaviour for the users.